templates/main.tftpl

```diff
@@ -1 +1,34 @@
-#This file is where the code for your application will be in the future.
\ No newline at end of file
+resource "azurerm_resource_group" "network" {
+  name     = "rg-$${local.application_name}Network-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+
+resource "azurerm_resource_group" "backup" {
+  name     = "rg-$${local.application_name}Backup-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
```

Comment:
This one will be created automatically by a policy if Backup is required for a VM. I think it would probably be best to not manage this RG via terraform, since the resources inside it wouldn't be managed by terraform either

Reply:
Deleted